.github/workflows/windows.yml

```diff
@@ -149,6 +147,8 @@ jobs:
               ;;
             ruby)
               MORE_MSYS_PKGS+=" mingw-w64-x86_64-ruby"
+              # MinGW-w64 with iostream need static link
+              echo "EXAMPLE_LIBS=-static-libstdc++ -static-libgcc" >> $GITHUB_ENV
```

Comment:
This doesn't seem right. Do all the tests fail with this? What's the error message. It's been a while, but I used mingw gcc to run ruby tests with configure.ac as it is now. Maybe a couple of tests require iostreams now???

Reply:
I saw it first with MingW-w64 Python.
The error is DLL loadd error.
Windows load API do not pass the actual error.
But perhaps the Windows logger or using a proper debugger.
As I do not use Windows myself, it is hard to find the real error.
Yet using static link with stdc++ bay-pass.
I do remind, the flags are for the testing.

If you wish to investigate or open a ticket, you're welcomed.